### PR TITLE
[libc] Add MPFR testing infra for float128.

### PR DESCRIPTION
**Pull request by lntue**

### CODE REVIEW

File: libc/src/__support/complex_type.h [hunk 1]
```diff
@@ -62,16 +61,15 @@ template <> struct make_real<cfloat16> {
...
+#ifdef LIBC_TYPES_CFLOAT128_IS_NOT_COMPLEX_LONG_DOUBLE
 template <> struct make_real<cfloat128> {
   using type = float128;
 };
 #endif
 
 template <typename T> using make_real_t = typename make_real<T>::type;
 
-template <typename T> LIBC_INLINE constexpr T conjugate(T c) {
+template <typename T> LIBC_INLINE static constexpr T conjugate(T c) {
```
Comment (nickdesaulniers) [inline libc/src/__support/complex_type.h:72]: If `conjugate` is already in a namespace, does it still need `static` linkage?

**Review by nickdesaulniers — APPROVED** [review]

File: libc/src/__support/complex_type.h (hunk 1 above)
Comment (SchrodingerZhu) [reply libc/src/__support/complex_type.h:72]: Why do we need to make conj to be local to the TU?

Comment (lntue) [reply libc/src/__support/complex_type.h:72]: I was just playing around to see if this symbol is exposed in the static archives and object files or not.  Reverted it back to before.

**Review by SchrodingerZhu — APPROVED** [review]

File: libc/src/__support/complex_type.h (hunk 1 above)
Comment (nickdesaulniers) [reply libc/src/__support/complex_type.h]: I think more generally, we can probably remove this pattern throughout the codebase; I've been noticing that we're not super consistent.

**Pull request merged**